packages/data-store/src/configs.ts

```diff
@@ -1,74 +1,27 @@
-import {DataStoreConfigs, SDKConfigs} from './types/types';
-import {getDataSource} from './typeorm';
-import {SDKConfigsEntity} from './typeorm/entities/sdk-configs.entity';
-
-export async function getSDKConfigs(
-  options: DataStoreConfigs,
-): Promise<SDKConfigs> {
-  let result = await getDataSource().getRepository(SDKConfigsEntity).find();
-
-  if (result.length === 1) {
-    return result[0];
-  }
-
-  if (result.length > 1) {
-    throw new Error('More than one SDKConfigsEntity found');
-  }
-
-  const sdkConfigs = {
-    id: 'sdk-configs',
-    activeWallet: options.sdkConfigs.activeWallet,
-    version: null,
-  };
-
-  await getDataSource().getRepository(SDKConfigsEntity).save(sdkConfigs);
-
-  return sdkConfigs;
-}
-
-export const defaultDataStoreConfigs: DataStoreConfigs = {
-  sdkConfigs: {
-    activeWallet: 'mainnet',
-  },
+import {DataStoreConfigs} from './types';
+import {genericDocumentNetworkResolver} from './document-network-resolver';
+
+export const DEFAULT_CONFIGS: DataStoreConfigs = {
+  defaultNetwork: 'mainnet',
+  nonIsolatedNetworks: false,
```

Comment:
what is this flag used for?

Reply:
Let me remove this flag, we are not using that. I was thinking about the possibility to allow handling documents from multiple networks in one wallet, the same approach we have now, the queries would not be scoped by the network, but for now, that's not a use case